### DISCUSSION TITLE
Can we render the content of preview (antd Image component) in some specific container?

### DISCUSSION THREAD

**Discussion by TigranHovsepyan** [ant-design/ant-design#42840] [Q&A]
I need to implement the functionality which requires to use Antd Image component but the preview should be looked like in screen. Any ways how I can achieve such behavior because now preview renders only in popup (whole screen), so my goal to do the same but in my specific container.
<img width="963" alt="Снимок экрана 2023-06-05 в 15 17 46" src="https://github.com/ant-design/ant-design/assets/18684407/f3af1e9c-a099-4c96-8510-ff95ef3d57bf">

**Comment by kiner-tang** [comment]
Maybe you can try to use `getContainer `: https://ant.design/components/image#previewtype

**Reply by TigranHovsepyan** [reply]
Unfortunately while I'm adding the getContainer for preview config it still displaying the preview in the whole screen window and not in container which I'm specified :( 

`const containerRef = useRef(null);`
```
<div ref={containerRef}>
  <Image
    src="https://dummyimage.com/600x400/000/fff&text=Test"
    width="80%"
    preview={{ getContainer: () => containerRef.current }}
    rootClassName="documentsBlock"
  />
</div>
```

**Reply by kiner-tang** [reply]
Yes, it seems to be a bug, let me try to fix it.

**Reply by kiner-tang** [reply]
After our discussion, getContainer will only insert the dom of the preview box into the target container, but still keep the full screen display, which is a better experience. For your original requirements, you may need to implement your own custom preview component to support your functionality.

**Reply by tigrago** [reply]
Ok, got you. Anyway, thank you very much for your support!
